jarbas/chamber_of_deputies/fields.py

```diff
@@ -1,6 +1,19 @@
+import json
+
 from rows import fields
 
 
+class FloatField(fields.FloatField):
+
+    @classmethod
+    def deserialize(cls, value, *args, **kwargs):
+        try:  # Rows cannot convert values such as '14,96' to float
+            value = float(value.replace(',', '.'))
```

Comment:
What if `value == 1.234,56`?

Reply:
This is a valid claim indeed. But as we kind of control the input data (via `serenata-toolbox`) we can assure there's no thousands separator. But this would be a good issue for first time contributors IMHO ; )

Reply:
At least at this moment of time/git tree, this case won't be triggered. The file is generated by the default Pandas csv exporter, which does not add a thounsands separator. [/serenata_toolbox/chamber_of_deputies/reimbursements_cleaner.py#L149](https://github.com/okfn-brasil/serenata-toolbox/blob/57c85dae6c5c825059ccd7cbee397f1fa4e45bba/serenata_toolbox/chamber_of_deputies/reimbursements_cleaner.py#L149)